MessageStackView/Views/Badges/BadgeContainerView.swift

```diff
@@ -15,7 +15,7 @@ import UIKit
 open class BadgeContainerView: UIView {
     
     /// Overrideable default shadow components for `BadgeContainerView`
-    static var shadowComponents: UIView.ShadowComponents = .default
+    public static var shadowComponents: UIView.ShadowComponents = .default
```

Comment:
@simonmitchell Was this intended?

Reply:
Making it internal? Definitely should be public yeah! Not sure how that happened!